test/NinjaTools.FluentMockServer.API.Tests.Exclude/Infrastructure/ScopedHttpContextRepository.cs

```diff
@@ -23,23 +23,23 @@ public MiddlewareTestBase(ITestOutputHelper output) : base(output)
     }
 
     [ExampleData(ExampleData = typeof(HttpContextExamples))]
-    [Story(   
+    [Story(
         Title = "Can store and retrieve data inside a scoped HttpContext",
         AsA = "As a user of the MockServer-API",
         IWant = "I want to setup up properties for a request",
         SoThat = "So that the MockServer later is setup up as expected."
-     )]
+    )]
     public class ScopedHttpContextRepositoryTests
     {
+        private readonly IHttpContextAccessor _contextAccessor;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_contextAccessor' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4739715)